src/utils/graphql/apollo.ts

```diff
@@ -0,0 +1,39 @@
+import { ApolloClient, from, HttpLink, InMemoryCache, type NormalizedCacheObject } from '@apollo/client/core';
+import { onError } from '@apollo/client/link/error';
+import { InternalServerErrorException } from '@utils';
+
+const errorLink = onError(({ graphQLErrors, networkError }) => {
+  if (graphQLErrors !== undefined) {
+    const errors = graphQLErrors.map(({ message, extensions, path }) => ({
+      message,
+      path: path?.map((path) => path).toString(),
+      extensions: JSON.stringify(extensions),
+    }));
+    throw new InternalServerErrorException('GraphQL Error', errors);
+  }
+  if (networkError !== undefined) {
+    throw new InternalServerErrorException(`GraphQL network error: ${networkError?.message}`);
+  }
+});
+
+const createHttpLink = (apiKey: string): HttpLink => {
+  return new HttpLink({
+    uri: process.env.LINEAR_GRAPHQL_URL!,
```

Comment:
Please, add LINEAR_GRAPH_URL to .env.template, and if necessary add it to the method that validates all env variables are present